### PR TITLE
Add initial web parts files

### PR DESCRIPTION
**Pull request by twsouthwick**
A couple of key integration points with page have been if/def'd out with a logging message if possible, otherwise an exception will be thrown for now.

### CODE REVIEW

**Comment by twsouthwick** [comment]
@birojnayak There's an app I want to port that needs this. Looks like it's stuck on a similar state of "given a type name, give me a type" as the user control.

Looks like we want this abstraction to be available per request as well as during the compilation. I'll see if I can have a proposal for that soon

**Comment by twsouthwick** [comment]
FYI as with most of these initial code dumps, I've taken care to only address compile warnings with either `#if PORT_{SomeIdentifier}` or not including the code with minimal formatting/code changes as required by our analyzers

File: src/WebForms/UI/TemplateControl.cs [hunk 1]
```diff
@@ -502,6 +504,20 @@ private MethodInfo GetInstanceMethodInfo(Type delegateType, string methodName)
...
 
+    internal Control LoadControl(string virtualPath)
+    {
+        Logger.LogWarning("Trying to load control for '{Path}'", virtualPath);
+        return new LoadedControl(virtualPath);
+    }
+
+    private sealed class LoadedControl(string virtualPath) : Control
+    {
+        protected internal override void Render(HtmlTextWriter writer)
```
Comment (birojnayak) [inline src/WebForms/UI/TemplateControl.cs:515]: Are you going to stitch this in later ?

**Review by birojnayak — APPROVED** [review]

File: src/WebForms/UI/TemplateControl.cs (hunk 1 above)
Comment (twsouthwick) [reply src/WebForms/UI/TemplateControl.cs:515]: yup

**Pull request merged**